src/css/layout/footer.scss

```
@@ -1,2 +1,64 @@
-@use '../object/utility/utility.scss';
-// footerのスタイルを記述するファイル
\ No newline at end of file
+@use '../object/utility/utility.scss' as base;
+// footerのスタイルを記述するファイル
+
+.p-footers {
```

[must]

> クラス名とscssファイル名が一致していること

https://gizumo.backlog.com/alias/wiki/2174119#:~:text=%E3%82%AF%E3%83%A9%E3%82%B9%E5%90%8D%E3%81%A8scss%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E5%90%8D%E3%81%8C%E4%B8%80%E8%87%B4%E3%81%97%E3%81%A6%E3%81%84%E3%82%8B%E3%81%93%E3%81%A8

という規約があるので、その規約に則って命名をするようにお願いします。
HTML 要素に付与している class 名からそのスタイルが定義されているファイルを特定しやすくするためにこちらの規約を設けています。
他のファイルについても見直してみてください。

こちらのセレクタは以下のようになるかと思います。

```suggestion
.l-footer {
```

---

クラス名を再定義しました。